### DISCUSSION TITLE
libsql on fedora 36

### DISCUSSION THREAD

**Discussion by tjyang** [tursodatabase/libsql#54] [Show and tell]
Hi All

- I (not familiar with Ubuntu) am reporting rpm pkgs  I missed when running rusttest , it got added into Makefile recently.

```
sudo dnf install -y rust-clang-sys+clang_9_0-devel sqlite-devel
```
- Successful result.

```
[me@ipa01 libsql]$ make rusttest;date;cat /etc/redhat-release
( cd test/rust_suite; cargo test )
    Finished test [unoptimized + debuginfo] target(s) in 0.06s
     Running unittests src/lib.rs (target/debug/deps/libsql_rust_suite-8cee9dc3a7978c64)

running 1 test
test tests::test_insert_steven ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests libsql_rust_suite

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

Tue Oct 25 08:48:06 AM CDT 2022
Fedora release 36 (Thirty Six)
[me@ipa01 libsql]$

```

**Comment by psarna** [comment]
Thanks! The fact that you needed to install sqlite-devel actually implies that `make rusttest` doesn't test the locally built library as it should, but instead tries to use the system library. I already had a fix for that in a separate, larger pull request, but it makes more sense to just fix it immediately, therefore: https://github.com/libsql/libsql/pull/55
